examples/vision/video_classification.py

```diff
@@ -1,19 +1,25 @@
 """
-Title: Video Classification with a CNN-RNN Architecture
-Author: [Sayak Paul](https://twitter.com/RisingSayak)
+Video Classification with a CNN-RNN Architecture
+Modified, corrected, and updated: [Mikolaj Buchwald](https://www.linkedin.com/in/mikolaj-buchwald/)
+Author of the original example: [Sayak Paul](https://twitter.com/RisingSayak)
 Date created: 2021/05/28
-Last modified: 2021/06/05
+Last modified: 2023/05/01
 Description: Training a video classifier with transfer learning and a recurrent model on the UCF101 dataset.
 Accelerator: GPU
 """
 """
 This example demonstrates video classification, an important use-case with
 applications in recommendations, security, and so on.
+
 We will be using the [UCF101 dataset](https://www.crcv.ucf.edu/data/UCF101.php)
 to build our video classifier. The dataset consists of videos categorized into different
 actions, like cricket shot, punching, biking, etc. This dataset is commonly used to
 build action recognizers, which are an application of video classification.
 
+The subset (subsampling) of the UCF-101 dataset that we will be working on is available
+at: https://zenodo.org/record/7882861#.ZE-UTC1Bxqs and will be downloaded using the `wget`
+command. On macOS, install the `wget` using `brew install wget`.
```

Comment:
We prefer downloading archives using [`tf.keras.utils.get_file()`](https://www.tensorflow.org/api_docs/python/tf/keras/utils/get_file).

Reply:
I will correct that as well, and I will test if fetching data this way will work

Reply:
I used [tf.keras.utils.get_file()](https://www.tensorflow.org/api_docs/python/tf/keras/utils/get_file), as you asked, thank you for showing me this function, it is quite nice. I had to make some further code modifications (regarding file path), as the file is downloaded at `~/.keras/...`, please revise.